extensions/kind/src/create-cluster.ts

```diff
@@ -79,13 +80,18 @@ export async function createCluster(
   // ok we need to write the file
   await fs.promises.writeFile(tmpFilePath, kindClusterConfig, 'utf8');
 
+  telemetryLogger.logUsage('createCluster', { provider, httpHostPort, httpsHostPort });
+
   // now execute the command to create the cluster
   const result = await runCliCommand(kindCli, ['create', 'cluster', '--config', tmpFilePath], { env, logger });
 
   // delete temporary directory/file
   await fs.promises.rm(tmpDirectory, { recursive: true });
 
   if (result.exitCode !== 0) {
+    telemetryLogger.logError('createCluster', { provider, error: result.error, stdErr: result.stdErr });
     throw new Error(`Failed to create kind cluster. ${result.error}`);
+  } else {
+    telemetryLogger.logUsage('createClusterSuccess', { provider });
```

Comment:
Why send a separate event for createCluster and createClusterSuccess? Then you need to correlate to see success %age. I was expecting a single post-event with e.g. { provider, httpHostPort, httpsHostPort, success/fail boolean }. (and still logging error if applicable)

Reply:
logging the error is on the logError callback

Reply:
Yes, but today there are two events in either path (create + success vs create + error) and if you want to check something like success rate you need to correlate two types of events. In the future if we wanted to do something like adding an attribute for how long it takes we'd have to log it in both success and error events and analyze these separately.

Always sending the createCluster after the action with an extra boolean would mean one less event on the success path (post-create vs post-create + error) and you'd only need to look at the one event unless you were interested in failures.

Reply:
I've used a single event (with error or usage)